synapse/lib/stormtypes.py

```diff
@@ -3933,6 +3933,16 @@ class LibQueue(Lib):
                   'returns': {'type': 'list',
                               'desc': 'A list of queue definitions the current user is allowed to interact with.', }}},
     )
+    _storm_lib_perms = (
+        {'perm': ('queue', 'add'), 'gate': 'cortex',
+         'desc': 'Permits a user to create a named queue.'},
+        {'perm': ('queue', 'get'), 'gate': 'queue',
```

Comment:
Do we have two separate gates for queue.get? It looks like there is a `queue:{name}` gate and also a `gateiden` gate??

Reply:
The queues do not have random idens; `queue:{name}` is the name of the gate.

Reply:
ah gotcha